dace/transformation/passes/analysis/loop_analysis.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019-2024 ETH Zurich and the DaCe authors. All rights reserved.
```

Comment:
no documentation whatsoever in this file. What is it used for and who is it used by

Reply:
The main point of it will be loop-based data flow analysis passes (loop carry dependencies etc.). Due to that, there are a few utility functions related to loop information that all loop-centric passes need in here. This is for now the only thing in here, but the file will be populated with more soon. I have added a doc comment for the current functionality, and will update with the coming additions.